crates/diesel_models/src/payouts.rs

```diff
@@ -29,6 +29,7 @@ pub struct Payouts {
     pub created_at: PrimitiveDateTime,
     #[serde(with = "common_utils::custom_serde::iso8601")]
     pub last_modified_at: PrimitiveDateTime,
+    pub attempt_count: i16,
```

Comment:
Since attempt count wouldn't be negative unsigned integers could be used. Also u8 type range would be more than sufficient

Reply:
Since diesel doesn't recognize u8, this will throw an error.